applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -51,9 +51,9 @@ endif::[]
 To create applications using the *Developer* perspective ensure that:
 
 * You have xref:../../web_console/web-console.adoc#web-console[logged in to the web console].
-* You are in the xref:../../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
 * You have created a project or have access to a project with the appropriate xref:../../authentication/using-rbac.adoc#default-roles_using-rbac[roles and permissions] to create applications and other workloads in {product-title}.
 
+
```

Comment:
Looks like you added an extra space here. It will not break anything, but its not needed here.